x-pack/plugin/core/template-resources/src/main/resources/kibana-reporting-template.json

```diff
@@ -0,0 +1,177 @@
+{
+  "index_patterns": [".kibana-reporting*"],
+  "priority": 100,
+  "data_stream": {
+    "hidden": true
+  },
+  "allow_auto_create": true,
+  "template": {
+    "settings": {
+      "number_of_shards": 1,
```

Comment:
I don't know much about how this data stream is used, but we don't want an ILM policy or data stream lifecycle? Does something in kibana clean this up periodically?

Reply:
We'll need something, since we don't yet have the transformation-on-Serverless-or-Serverfull piece, I figured we would handle it in a follow-up